### PR TITLE
added shell cleaning trick

### PR DESCRIPTION
**Pull request by caverav**
I think that this trick could help some pentesters to get a comfortable shell.

### CODE REVIEW

**Review by Sleekbobby1011 — APPROVED** [review]

**Review by bobcampbell112233 — COMMENTED** [review]
Veery good tools

**Review by Theanulbeed28 — COMMENTED** [review]
[Dalvik](https://developers.facebook.com/docs/graph-api)

**Comment by Tingsook1981** [comment]
4507af97cbb5a31d948c58a62980f268e0054f03

**Comment by Spacesugarog** [comment]
- [ ] uj u u

**Review by Daddyzilla65 — APPROVED** [review]

**Review by Shinebest123 — COMMENTED** [review]
Ghj

**Review by TharealKey — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -114,8 +114,9 @@ Only main chapters:
 - **[Hacking/Penetration Testing](#hackingpenetration-testing-toc)**
 - **[Your daily knowledge and news](#your-daily-knowledge-and-news-toc)**
 - **[Other Cheat Sheets](#other-cheat-sheets-toc)**
-- **[One-liners](#one-liners-toc)**
-- **[Shell functions](#shell-functions-toc)**
+- **[Shell One-liners](#shell-one-liners-toc)**
```
Comment (Livingstone110) [inline README.md:117]:
```suggestion
- **[Shell One-liners](#shell-one-liners-toc)**
```

Comment (Livingstone110) [inline README.md:117]: README.md

File: README.md [hunk 2]
```diff
@@ -2327,15 +2328,17 @@ ___
 ```bash
 vimdiff file1 file2
 ```
+
```
Comment (Livingstone110) [inline README.md:2331]: README.md

File: README.md (hunk 1 above)
Comment (Livingstone110) [reply README.md:117]: README.md

Comment (Dirtyfaygo) [reply README.md:117]: Nice

**Comment by Gerrime** [comment]
Boost volume

**Comment by porterjb** [comment]
d30037f1671a853ab91a3e43a78c2f048b7b3ca2

**Review by Lukasog222 — COMMENTED** [review]
README.md

File: README.md (hunk 1 above)
Comment (Lukasog222) [inline README.md:117]: README.md

**Review by Skunkyducker — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -114,8 +114,9 @@ Only main chapters:
 - **[Hacking/Penetration Testing](#hackingpenetration-testing-toc)**
 - **[Your daily knowledge and news](#your-daily-knowledge-and-news-toc)**
 - **[Other Cheat Sheets](#other-cheat-sheets-toc)**
-- **[One-liners](#one-liners-toc)**
-- **[Shell functions](#shell-functions-toc)**
+- **[Shell One-liners](#shell-one-liners-toc)**
+- **[Shell Tricks](#shell-tricks-toc)**
+- **[Shell Functions](#shell-functions-toc)**
```
Comment (Trujillo1G) [inline README.md:119]: Ok

File: README.md [hunk 4]
```diff
@@ -1669,7 +1670,7 @@ _Encoding_
 text :arrow_right: encoded<br>
 text :arrow_left: encoded
 
-#### One-liners &nbsp;[<sup>[TOC]</sup>](#anger-table-of-contents)
```
Comment (Trujillo1G) [inline README.md:1672]: Huh

File: README.md (hunk 1 above)
Comment (Trujillo1G) [reply README.md:117]: Ok

Comment (Trujillo1G) [reply README.md:117]: Ok> README.md

**Review by kiseCrow — COMMENTED** [review]
Hello, can I use your code?

**Comment by Trujillo1G** [comment]
Negative I don’t know what’s it for or anything.

File: README.md (hunk 4 above)
Comment (Trujillo1G) [reply README.md:1672]: Ok

**Review by Dreday10 — APPROVED** [review]

**Review by Masw21 — COMMENTED** [review]
Ok> README.md

**Review by Masw21 — COMMENTED** [review]
Ok> README.Md

**Review by Scutua — APPROVED** [review]

**Review by Sholman81 — COMMENTED** [review]
ty

**Comment by Hostitel** [comment]
Ok👍

**Review by Jorge-Martin777 — COMMENTED** [review]
I approve

File: README.md [hunk 5]
```diff
@@ -114,8 +114,9 @@ Only main chapters:
 - **[Hacking/Penetration Testing](#hackingpenetration-testing-toc)**
 - **[Your daily knowledge and news](#your-daily-knowledge-and-news-toc)**
 - **[Other Cheat Sheets](#other-cheat-sheets-toc)**
-- **[One-liners](#one-liners-toc)**
-- **[Shell functions](#shell-functions-toc)**
```
Comment (Jorge-Martin777) [inline README.md:118]: #shell function sudo close all ports, front and end toc

**Review by Jorge-Martin777 — APPROVED** [review]
Sudo

**Review by Jorge-Martin777 — CHANGES_REQUESTED** [review]
Update Read

**Review by maxpayne66613 — APPROVED** [review]

**Review by boltenstanley-hub22 — APPROVED** [review]

**Review by Lmillan12co — APPROVED** [review]

**Review by Lmillan12co — APPROVED** [review]

**Review by chrisboley53-debug — APPROVED** [review]

**Comment by boltenstanley-hub22** [comment]
Okay so how do we go ahead

**Review by Lmillan12co — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (LaColombe210) [reply README.md:117]: Q: how do I find things linked to me

File: README.md (hunk 5 above)
Comment (mabdollahi1379-arch) [reply README.md:118]: q

Comment (boltenstanley-hub22) [reply README.md:118]: #execute

**Review by teremuhamblin — APPROVED** [review]

**Review by mabdollahi1379-arch — COMMENTED** [review]
tnx

**Review by mabdollahi1379-arch — COMMENTED** [review]
tnxs

**Review by sowja293 — COMMENTED** [review]
May beee

**Review by drpsykkjoker — APPROVED** [review]

**Review by corkumandrew557-tech — APPROVED** [review]

**Review by UnknownNameL7 — APPROVED** [review]

**Review by lopezregalodelo-a11y — COMMENTED** [review]
Updated

**Review by lopezregalodelo-a11y — APPROVED** [review]
Updated

**Review by lopezregalodelo-a11y — CHANGES_REQUESTED** [review]
Updated

**Comment by boltenstanley-hub22** [comment]
Okay

**Comment by boltenstanley-hub22** [comment]
Rub a tiny amount of mineral oil or baby oil after it's completely dry. It
really brings back the color.

**Pull request merged**